crates/cairo-coverage/src/input/filter.rs

```diff
@@ -0,0 +1,90 @@
+use crate::cli::IncludedComponent;
+use crate::data_loader::LoadedData;
+use crate::input::sierra_to_cairo_map::StatementOrigin;
+use regex::Regex;
+use std::collections::HashSet;
+use std::iter::once;
+use std::sync::LazyLock;
+
+pub static VIRTUAL_FILE_REGEX: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"\[.*?]").unwrap());
+const SNFORGE_TEST_EXECUTABLE: &str = "snforge_internal_test_executable";
+
+#[derive(Eq, PartialEq, Hash)]
+enum ItemLabel {
```

Comment:
The name `ItemLabel` seems completely off to me. Is there any valid reason for using such a name? Wouldn't `CodeCategory` or something similar be more suitable?